src/cljs/main/broadfcui/page/workspace/notebooks/cluster_details.cljs

```diff
@@ -0,0 +1,44 @@
+(ns broadfcui.page.workspace.notebooks.cluster_details
+  (:require
+   [dmohs.react :as react]
+   [broadfcui.common.components :as comps]
+   [broadfcui.components.modals :as modals]
+   [broadfcui.components.spinner :refer [spinner]]
+   [broadfcui.endpoints :as endpoints]
+   [broadfcui.utils.ajax :as ajax]
+   ))
+
+(react/defc ClusterDetailsViewer
+  {:component-did-mount
+   (fn [{:keys [this]}]
+     (this :-get-cluster-details))
+   :render
+   (fn [{:keys [state this props]}]
+     (let [{:keys [server-response cluster-details]} @state
+           {:keys [server-error]} server-response]
+       [modals/OKCancelForm
+        {:header "Cluster Details"
+         :dismiss (:dismiss props)
+         :ok-button {:text "Close" :onClick (:dismiss props)}
+         :show-cancel? false
+         :content
+         (react/create-element
+          [:div {}
+           [comps/ErrorViewer {:error server-error}]
+           (if cluster-details
+             (str cluster-details) ;TODO display in a nicer way than just raw json
```

Comment:
This is still a TODO -- I'd like to display cluster details in sort of a tabular way.